algorithms/Path/maxflow/tg_maxflow.gsql

```diff
@@ -0,0 +1,186 @@
+CREATE QUERY tg_maxflow(VERTEX source, VERTEX sink, Set<STRING> v_type, SET<STRING> e_type, 
+       SET<STRING> reverse_e_type, STRING cap_attr, STRING cap_type, FLOAT min_flow_threshhold = 0.001, 
+       BOOL print_accum = TRUE, BOOL display_edges = TRUE, BOOL spit_to_file = FALSE, 
+       STRING file_path = "/home/tigergraph/tg_query_output.csv"){ 
+  
+  /*
+   Maxflow algorithm. Finds the maximum amount of flow that source can pushes to sink
+
+   source: start vertex to  every other vertex.     cap_type: capacity data type (UINT, INT,FLOAT,DOUBLE)
+   sink:   end vertex                               min_flow_threshhold: smallest epsilon for flow
+   v_type: vertex types to traverse                 print_accum: print JSON output
+   e_type: edge types to traverse                   display_edges: output edges for visualization
```

Comment:
comment for `display_edges` should clarify this only works when `print_accum = TRUE`